fluxcd/pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-fluxcd"
+description = "The fluxcd check"
+readme = "README.md"
+license = {text = "BSD-3-Clause"}
+keywords = [
+    "datadog",
+    "datadog agent",
+    "datadog check",
+    "fluxcd",
+]
+authors = [
+    { name = "Datadog", email = "packages@datadoghq.com" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: System :: Monitoring",
+    "Private :: Do Not Upload",
+]
+dependencies = [
+    "datadog-checks-base>=25.1.0",
```

Comment:
I think you need at least 32.6.0 because pydantic

Reply:
Shouldn't that have happened in [this PR](https://github.com/DataDog/integrations-extras/pull/2088/files)?

Reply:
It should have

Reply:
Oh, I guess we missed some, cuz here's [another PR](https://github.com/DataDog/integrations-extras/pull/2184/files) that's bumping the base check dependency **way after the pydantic v2 migration**